invisible_cities/reco/paolina_functions_test.py

```diff
@@ -209,3 +209,27 @@ def test_voxelize_single_hit():
     hits = [BHit(1, 1, 1, 100)]
     vox_size = np.array([10,10,10], dtype=np.int16)
     assert len(voxelize_hits(hits, vox_size)) == 1
+
+def test_length():
+    voxel_spec = ((10,10,10,  1000),
+                  (10,10,11,     1),
+                  (10,10,12,     2),
+                  (10,10,13,     4),
+                  (10,10,14,     8),
+                  (10,10,15,    16),
+                  (10,11,15,    32),
+                  (10,12,15,    64),
+                  (10,13,15,   128),
+                  (10,14,15,   265),
+                  (10,15,15,   512)
+    )
+    voxels = [Voxel(x,y,z, E) for (x,y,z,E) in voxel_spec]
+    tracks  = make_track_graphs(voxels, np.array([1,1,1]), contiguity=1.85)
+
+    assert len(tracks) == 1
+    distances = shortest_paths(tracks[0])
+    track_length = length(distances)
+
+    expected_length = 8 + np.sqrt(2)
+
+    assert_almost_equal(track_length, expected_length)
```

Comment:
As a general point for @everybody-in-IC, I recommend using [`pytest.approx`](https://docs.pytest.org/en/latest/builtin.html#comparing-floating-point-numbers) for doing float comparisons. It now supports numpy arrays (and other containers)!